storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/TopicBasedRemoteLogMetadataManagerConfig.java

```diff
@@ -53,7 +53,7 @@ public final class TopicBasedRemoteLogMetadataManagerConfig {
     public static final short DEFAULT_REMOTE_LOG_METADATA_TOPIC_REPLICATION_FACTOR = 3;
     public static final long DEFAULT_REMOTE_LOG_METADATA_CONSUME_WAIT_MS = 2 * 60 * 1000L;
     public static final long DEFAULT_REMOTE_LOG_METADATA_INITIALIZATION_RETRY_MAX_TIMEOUT_MS = 2 * 60 * 1000L;
-    public static final long DEFAULT_REMOTE_LOG_METADATA_INITIALIZATION_RETRY_INTERVAL_MS = 5 * 1000L;
+    public static final long DEFAULT_REMOTE_LOG_METADATA_INITIALIZATION_RETRY_INTERVAL_MS = 1000L;
```

Comment:
Nice catch! In our internal branch, it's been configured to 100 ms.

Reply:
Nice catch!

We can update this default value now without making any announcement as it is the first release we are releasing these configs for users/devs. 

@showuon @kamalcph Should we reduce it to 100ms instead of 1 sec?

Reply:
Sounds good to me. PR updated to 100ms by default.